test/unit/presenters/formats/completed_transaction_presenter_test.rb

```diff
@@ -75,7 +75,6 @@ def result
     edition.update_attribute(:slug, 'foo')
     expected = [
       { path: '/foo', type: 'prefix' },
```

Comment:
If it's worth doing, for the sake of neatness it might be worth keeping the trailing comma in the other examples then..